core/build.gradle

```diff
@@ -94,8 +94,8 @@ dependencies {
       exclude group: 'org.elasticsearch', module: 'elasticsearch'
     }
   }
-  testCompile 'com.google.jimfs:jimfs:1.1'
-  testCompile 'com.google.guava:guava:18.0'
+  testCompile "com.google.jimfs:jimfs:${versions.jimfs}"
+  testCompile "com.google.guava:guava:${versions.guava}"
```

Comment:
ok, fair enough. I can only say that it works w.r.t. API's, haven't done any intensive testing.